Mathlib/Data/ENat/Lattice.lean

```diff
@@ -85,6 +86,20 @@ lemma sSup_eq_zero' : sSup s = 0 ↔ s = ∅ ∨ s = {0} :=
 @[simp] lemma iSup_eq_zero : iSup f = 0 ↔ ∀ i, f i = 0 := iSup_eq_bot
 @[simp] lemma iSup_zero : ⨆ _ : ι, (0 : ℕ∞) = 0 := by simp
 
+lemma exists_eq_iInf [Nonempty ι] (f : ι → ℕ∞) : ∃ a, f a = ⨅ x, f x := by
+  suffices aux : ∀ f' : PLift ι → ℕ∞, ∃ a, f' a = ⨅ x, f' x by
```

Comment:
I'm guessing this is needed to allow iota to be a Prop too; but is this separate case a sign that something else has missing API?

Reply:
I was worried about that particular rabbit-hole, but it was shockingly easy to fix! All it took was changing `variable {ι : Type*}` to `variable {ι : Sort*}` at the top of `CompleteLinearOrder.lean`.